modules/backend/console/CreateController.php

```diff
@@ -69,6 +67,13 @@ protected function prepareVars(): array
             $model = Str::singular($vars['name']);
         }
         $vars['model'] = $model;
+        $vars['body_class'] = $this->option('sidebar') ? 'compact-container' : '';
+
+        $suffix = $this->option('sidebar') ? '-sidebar' : '';
+
+        $this->stubs["scaffold/controller/create{$suffix}.stub"] = 'controllers/{{lower_name}}/create.php';
+        $this->stubs["scaffold/controller/update{$suffix}.stub"] = 'controllers/{{lower_name}}/update.php';
+        $this->stubs["scaffold/controller/preview{$suffix}.stub"] = 'controllers/{{lower_name}}/preview.php';
```

Comment:
Can we have the stubs be in a folder by style / layout? i.e: `controller/[standard|sidebar]/create.stub`

Reply:
That would mean duplicating all the other stubs that stay the same for both layouts?

Reply:
Or do you mean only for the create/update/preview stubs?

Reply:
Just the layout specific ones